.gitlab-ci.yml

```diff
@@ -162,7 +162,25 @@ test:drupalcheck:
   dependencies:
   - deploy:review
 
-test:watchdog:
+test:statusreportval:
+  stage: tests
+  environment:
+    url: http://${CI_ENVIRONMENT_SLUG}.${REVIEW_DOMAIN}
+    name: review/$CI_COMMIT_REF_NAME
+    on_stop: stop_review
+  script:
+  - echo "Starting job script in ${BUILD_DIR}"
+  - cd ${BUILD_DIR}
+  - pwd
+  - ls -lah
+  - ls -lah scripts/makefile/
+  - make statusreportval
+  tags:
+  - review-apps-3
+  dependencies:
+  - deploy:review
+
+logs:watchdog:
```

Comment:
There's no logs stage defined for pipeline

Reply:
Yes there is https://github.com/skilld-labs/skilld-docker-container/blob/1102d0cf21b0d4521146af54e39ac2c985afc705/.gitlab-ci.yml#L14